nss-resolute-partridges/nutshell
# Raf Branch - Event Generator

## Link to Ticket
Closes #25 and #26

## Description of Proposed Changes
When user logs in, they will see an "Add Event" button
When user clicks "Add Event" they will be presented with 3 input fields for: Event Name, Event Location, Event Date
When user clicks "Submit Button" it will create the Event and push the information to the database
## Steps to Test
Outline the steps to test

```
git fetch --all
git checkout RafBranch
```

1. Run Build
2. Click "Add Event"
3. Input Event Name, Event Location, Event Date
4. Click submit button
5. Look at database to see if your event is being stored

## Impacted Areas in Application
List general components of the application that this PR will affect:

eventsGenerator.js
index.html
## Mentions @username
Tag users that need to review this code

## Definition of Done
The project must be fully documented. This includes the following:
Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
Every class must be documented with purpose, author, and methods.
Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
The project must be able to run fully, and without errors, on each teammate's system.
Fulfills every requirement.
Every line of code has been peer reviewed.
For projects that require unit testing, core functionality must be identified and have at least one test for each.
